app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -136,6 +142,13 @@ class PatientSummaryScreen(context: Context, attrs: AttributeSet) : RelativeLayo
         .subscribe { uiChange -> uiChange(this) }
   }
 
+  private fun enableEditPatientFeature() {
```

Comment:
🙀 Let's do this from the controller and avoid blocking Rx calls?

Reply:
The reason I didn't put this in the controller was because this is a throwaway flag that will be removed as soon as the edit patient feature is done. I don't have a problem moving it, however.

Reply:
I see you've fixed this.